### ISSUE TITLE
`shadow` not found... but it's listed as an alternative

### ISSUE THREAD

**Issue by sampotts** [ben-rogerson/twin.macro#216]
When upgrading to `2.0.2` and building our app, I get this:

![image](https://user-images.githubusercontent.com/719092/100541713-c582c980-3299-11eb-97d0-e5cf441b6b0d.png)

> ✕ shadow was not found
> 
> Try one of these classes:
> shadow-xs [0 0 0 1px rgba(0, 0, 0, 0.05)] / shadow-sm [0 1px 2px 0 rgba(0, 0, 0, 0.05)]
> shadow [0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06)]
> ...

Unless there's something funky going on I need to fix? It seems odd though.

**Comment by ben-rogerson** [comment]
Hey there,

[Check out this change](https://github.com/tailwindlabs/tailwindcss/pull/2580) that was made in Tailwind.

You'll need to add an item with the key name of `DEFAULT` in order to use the default value like you're used to.

For example:

To use `border`:

```js
// tailwind.config.js
borderWidth: {
      DEFAULT: "1px",
      0: "0",
      2: "2px",
      4: "4px",
      8: "8px",
},
```

I think I may need to adjust the suggestion in this case, I don't think it should be recommending the shadow class as being available.

**Comment by ben-rogerson** [comment]
Shadow classes actually still need to be updated to tailwind v2. Missed those somehow #224

**Comment by ben-rogerson** [comment]
Fixed the suggestions in v2.0.7 - cheers
